src/GitHub.App/Services/PullRequestEditorService.cs

```diff
@@ -336,6 +336,7 @@ public IVsTextView FindActiveView()
         /// <param name="fromLines">The document we're navigating from.</param>
         /// <param name="toLines">The document we're navigating to.</param>
         /// <param name="line">The 0-based line we're navigating from.</param>
+        /// <param name="matchLinesAbove"></param>
```

Comment:
`The number of lines to match`.

Reply:
@jcansdale shouldn't the default for this parameter be `MatchLinesAboveTarget` instead of `0`?